RetailCoder.VBE/Inspections/QuickFixes/PassParameterByReferenceQuickFix.cs

```diff
@@ -15,11 +15,13 @@ namespace Rubberduck.Inspections.QuickFixes
     public class PassParameterByReferenceQuickFix : QuickFixBase
     {
         private Declaration _target;
+        private AssignedByValParameterQuickFixHelper _quickFixHlpr;
```

Comment:
So...I'm going to change the names of the two AssignedByVal* QuickFIxes to be more precise.  I'll also change the helper class name to the more generic `QuickFixHelper.`  Over time, the hope is that the helper class disappears in favor of a future `CodPaneRewriter` class.